### PR TITLE
Many fixes

### PR DESCRIPTION
**Pull request by publicarray**

### CODE REVIEW

**Comment by publicarray** [comment]
Hi @stal888 would you mind taking a look?

**Comment by publicarray** [comment]
@robinlinden can this be merged? This is will also upgrade to libtoxcore to v0.2.x (the switch to cmake)

**Comment by robinlinden** [comment]
I don't have the rights to merge anything in this org. @nurupo merges things into tox.chat, so maybe they can do it.

File: Formula/libtoxcore@0.1.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
+class LibtoxcoreAT01 < Formula
+  desc "Tox library written in C"
+  homepage "https://tox.chat"
+  head "git://github.com/TokTok/c-toxcore", branch: "0.1.x"
```
Comment (robinlinden) [inline Formula/libtoxcore@0.1.rb]: Use the v0.1.11 tag instead. That branch was never released and should be deleted.

File: Formula/toxic.rb [hunk 2]
```diff
@@ -1,49 +1,47 @@
...
-  head "git://github.com/Tox/toxic", :using => :git
+  desc "An ncurses-based Tox client"
   homepage "https://tox.chat"
+  head "git://github.com/Tox/toxic"
 
-  depends_on "libtoxcore"
-  depends_on "ncurses"
+  option "with-notifications", "Build with desktop notifications"
+
+  depends_on "pkg-config" => :build
```
Comment (robinlinden) [inline Formula/toxic.rb]: Not sure how to dependencies inn Homebrew works, but these don't match the ones listed at https://github.com/JFreegman/toxic/blob/master/INSTALL.md#dependencies

(The Python one seems to be missing at least)

**Comment by nurupo** [comment]
@robinlinden I can give you rights on this repo. It's not being maintained and I was thinking of moving this repo away into https://github.com/Tox-Archive though, along with https://github.com/Tox/arch-repo-tox and https://github.com/Tox/arch-repo-tox, to make this GH org less of a graveyard.

**Comment by nurupo** [comment]
@robinlinden there you go, gave your permissions.

File: Formula/libtoxcore@0.1.rb (hunk 1 above)
Comment (publicarray) [reply Formula/libtoxcore@0.1.rb]: Thanks mate!

**Comment by publicarray** [comment]
@nurupo I understand your concern but I'm willing to provide support as time allows. I use this repo to make it easy and keep my libtoxcore and other dependencies up-to-date. (much faster and less error prone than cd some_dir && git reset --hard && git pull && git checkout some_tag && cmake . some_args)

**Comment by robinlinden** [comment]
@publicarray Done with the PR?

**Comment by publicarray** [comment]
@robinlinden I think so. Unless you can find any more issues, I think Its ready.

**Pull request merged**